homeassistant/components/hausbus/const.py

```diff
@@ -0,0 +1,7 @@
+"""Constants for the Haus-Bus integration."""
+import logging
+
+LOGGER = logging.getLogger(__package__)
+
+DOMAIN = "hausbus"
+ATTR_ON_STATE = "on_state"
```

Comment:
Only used in `light.py`, please move it there unless you will use it from other modules in a follow-up

Reply:
The idea was to also use it for other platforms like switch.

Reply:
Let's not worry about the future before we are there